devops/actions/run-tests/benchmark/action.yml

```diff
@@ -133,6 +133,11 @@ runs:
       cmake --install build
 
       cd -
+  - name: Install linux-tools package
+    shell: bash
+    run: |
+      sudo apt-get update
```

Comment:
does this action run inside docker and if so is the image one of the ones we create in this repo? if yes to both i would prefer we do this as part of image creation instead of in an action